packages/participants-svc/src/domain/participant_agg.ts

```diff
@@ -917,34 +916,182 @@ export class ParticipantAggregate {
             );
         }
 
-        let createdId: string;
-        try {
-            this._accBal.setToken(secCtx.accessToken);
-            createdId = await this._accBal.createAccount(
-                account.id,
-                participantId,
-                account.type,
-                account.currencyCode
+        existing.participantAccountsChangeRequest.push({
+            id: accountChangeRequest.id || randomUUID(),
+            accountId: accountChangeRequest.accountId,
+            type: accountChangeRequest.type as ParticipantAccountTypes,
+            currencyCode: accountChangeRequest.currencyCode,
+            creditBalance: null,
+            debitBalance: null,
+            balance: null,
+            externalBankAccountId: accountChangeRequest.externalBankAccountId,
+            externalBankAccountName: accountChangeRequest.externalBankAccountName,
+            createdBy: secCtx.username!,
+            createdDate: Date.now(),
+            approved: false,
+            approvedBy: null,
+            approvedDate: null
+        });
+        existing.changeLog.push({
+            changeType: ParticipantChangeTypes.CHANGE_ACCOUNT,
+            user: secCtx.username!,
+            timestamp: Date.now(),
+            notes: null,
+        });
+
+        const updateSuccess = await this._repo.store(existing);
+        if (!updateSuccess) {
+            const err = new CouldNotStoreParticipant(
+                "Could not update participant on addParticipantAccount"
             );
-        } catch (err) {
             this._logger.error(err);
-            if (err instanceof UnauthorizedError) throw err;
+            throw err;
+        }
+
+        this._logger.info(
+            `Successfully created account to Participant with ID: '${participantId}'`
+        );
+
+        await this._auditClient.audit(
+            AuditedActionNames.PARTICIPANT_ACCOUNT_CHANGE_REQUEST_CREATED,
```

Comment:
This should be ADD or CREATE, see names below

Reply:
Updated with ADD.